### DISCUSSION TITLE
Initial tracks, loop boundaries and default input

### DISCUSSION THREAD

**Discussion by andrasf** [helgoboss/helgobox#1296] [Q&A] [answered]
Hi,

I have three questions, I don't know if I should open a separate topic for each or not.

First is, when loading Playtime, already existing tracks are not shown. Will it be added or is there an obstacle for this? Would be very useful to be able to use it on already existing timeline projects.

Second, setting audio loop boundaries would be really useful. I know this is still in progress, just wanted to ask about the timeline plans on this.

Third, all my new tracks come with audio input set for them as default, most of the time first thing I have to do is changing it to midi. Is there a possibility to change this, so it comes with midi as default?

Thanks,
Andras

**Comment by helgoboss** [comment, marked as answer]
> Hi,
> 
> I have three questions, I don't know if I should open a separate topic for each or not.
> 
> First is, when loading Playtime, already existing tracks are not shown. Will it be added or is there an obstacle for this? Would be very useful to be able to use it on already existing timeline projects.

You can select the existing tracks, right-click a column header and then choose "Insert column for each selected track".

> 
> Second, setting audio loop boundaries would be really useful. I know this is still in progress, just wanted to ask about the timeline plans on this.

There's a way to control the section in the inspector (not yet in the waveform viewer). If you are not talking about the section but about the ability to define only a portion of the section to be looped, this is not yet in the works and is expected to be added after the launch.

> Third, all my new tracks come with audio input set for them as default, most of the time first thing I have to do is changing it to midi. Is there a possibility to change this, so it comes with midi as default?

Not directly. But you can use the usual REAPER means to add tracks as you want them (with your favorite defaults, track templates, project templates, etc.) and then use "Insert column for each selected track".

**Comment by andrasf** [comment]
Thank you, I missed the insert column for tracks option. Also got the track default inputs set correctly now.

For the audio I meant selecting the portion to be looped. In fact I used the Length setting but can't remember seeing the Start position setting from couple of days ago. Maybe I need better glasses :)

Thanks for the help.
